tasks/sep/020-holocene-system-config-upgrade-multi-chain/OVERVIEW.md

```diff
@@ -0,0 +1,68 @@
+# Holocene Hardfork - SystemConfig Upgrade
```

Comment:
I'd delete this file for simplicity—it's not required, and it's just another thing that could get out of date when making changes to this PR, or the equivalent mainnet PR. The `input.json` and resulting `VALIDATION.md` are the source of truth

Reply:
[4e2aa1a](https://github.com/ethereum-optimism/superchain-ops/pull/378/commits/4e2aa1a29a585c90885615f70af48de6cd800278)